install/Kubernetes/helmcharts/aciplugin/templates/configmaps.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: aciplugin-config
+  namespace: {{ .Values.odimra.namespace }}
+data: 
+  config.json: |
+    {
+    	"RootServiceUUID": {{ .Values.aciplugin.aciPluginRootServiceUUID | quote }},
+    	"PluginConf": {
+    		"ID": "ACI",
+    		"Host": {{ .Values.aciplugin.host | quote }},
+    		"Port": "45020",
+    		"UserName": {{ .Values.aciplugin.username | quote }},
+    		"Password": {{ .Values.aciplugin.password | quote }}
+    	},
+    	"EventConf": {
+    		"DestinationURI": "/redfishEventListener",
+    		"ListenerHost": {{ .Values.aciplugin.eventHost | quote }},
+    		"ListenerPort": "45021"
+    	},
+    	"KeyCertConf": {
+    		"RootCACertificatePath": "/etc/aciplugin_certs/rootCA.crt",
+    		"PrivateKeyPath": "/etc/aciplugin_certs/odimra_server.key",
+    		"CertificatePath": "/etc/aciplugin_certs/odimra_server.crt"
+    	},
+    	"TLSConf" : {
+    		"MinVersion": "TLS_1.2",
+    		"MaxVersion": "TLS_1.2",
+    		"VerifyPeer": true,
+    		"PreferredCipherSuites": [
+    			"TLS_RSA_WITH_AES_128_GCM_SHA256",
+    			"TLS_RSA_WITH_AES_256_GCM_SHA384",
+    			"TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
+    			"TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
+    			"TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+    			"TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384"
+    		]
+    	},
+    	"FirmwareVersion": "v1.0.0",
+    	"SessionTimeoutInMinutes": 30,
+    	"LoadBalancerConf": {
+    		"LBHost": {{ .Values.aciplugin.lbHost | quote }},
+    		"LBPort": {{ .Values.aciplugin.lbPort | quote }}
+    	},
+    	"MessageBusConf": {
+    		"MessageQueueConfigFilePath": "/etc/aciplugin_config/platformconfig.toml",
```

Comment:
this value need to be updated please check in  ilo plugin please make changes  accordingly